message_definitions/v1.0/common.xml

```diff
@@ -1817,31 +1816,31 @@
     </enum>
     <enum name="MAV_CMD_ACK">
```

Comment:
I wonder if this one should be deprecated?

Neither PX4 nor ardupilot seem to use this one, using `MAV_RESULT` instead.

Reply:
Can You do an upstream PR for it?